docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -0,0 +1,69 @@
+# Google Cloud - Overview
+
+[Overview](#overview)
+
+[Aims](#aims)
+
+[Initial stages](#initial-stages)
+
+[Next stages](#next-stages)
+
+[Links](#links)
+
+[Assets](#assets)
+
+## Overview
+
+![Google Cloud - Doubtfire](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_Doubtfire.jpg "Google Cloud - Doubtfire")
+
+The Google cloud team have deployed a functional instance of Doubtfire (also known as OnTrack)
+hosted on Google Cloud Platform (GCP).
+
+Our GCP project is centrally managed by Deakin IT where the team have been granted access using our
+Deakin Google Workspace accounts.
+
+- GCP Project Name: sit-22t1-ontrack-deplo-d026375
+- GCP Project ID: sit-22t1-ontrack-deplo-d026375
+
+In Google Compute Engine, we have created server instance running Linux (Centos 7.x). The instance
+is a small, initial footprint that has a publicly facing network and accessible over the internet
+via HTTPS (port 443).
+
+We have used the source code from the Thoth Tech repository for the deployment into GCP, where we
+used **docker compose** to deploy the images for the components required to run Doubtfire (api
+server, app server, doubtfire-web, mariadb, nginx).
+
+## Aims
+
+- Deploy a student-run version of OnTrack hosted on Google Cloud. This version will be running in
+  isolation, independently of the Deakin University version.
+- Create a CI/CD pipeline that automates the building, deployment, and validation of a of the Thoth
+  Tech OnTrack to Google Cloud.
+
+## Initial stages
+
+Initially, the Google Cloud team had spent time understanding GCP, Docker, and the Doubtfire
+deployment. We ran several tests locally prior to deploying a server instance in GCP and deploying
```

Comment:
Is there any documentation for the tests? How to run the test locally? or what tests have been run?

Reply:
I have expanded on the details and shared screenshots of each stage and how we went from testing locally (localhost on our workstations) to deploying into GCP.